### PR TITLE
Add requirement in getting started scalar manager

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR adds a requirement section in the Scalar Manager getting started guide.

Before, we pulled all images from GitHub Packages in all getting started guides. And, we created a secret resource that includes GitHub Personal Access Token (PAT) in the getting started guide of ScalarDB or ScalarDL.
So, we could pull the Scalar Manager container image using this PAT.

However, I updated the getting started guide of ScalarDB and ScalarDL in PR #147.
After that update, we pull ScalarDB/ScalarDL container images from AWS/Azure Marketplace.
So, we don't create a secret resource that includes PAT of GitHub.

In other words, there are no steps to create a secret resource that includes PAT to pull the Scalar Manager container image.
As a result, we will face the `Error: ErrImagePull` error when we deploy Scalar Manager according to the getting started.

So, I added a requirement section that describes how to create PAT to pull the Scalar Manager container image.
(We can remove this section after we start providing Scalar Manager in each Marketplace.)

Please take a look!

### CODE REVIEW

**Review by supl — APPROVED** [review]
LGTM.
Thank you!

File: docs/getting-started-scalar-manager.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Scalar Manager also embeds Grafana explorers by which the users can review the m
 This guide assumes that the users are aware of how to deploy Scalar products with the monitoring and logging tools to a Kubernetes cluster.
 If not, please start with [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) before this guide.
 
+## Requirement
+
+* You need the privileges to pull the Scalar Manager container (`scalar-manager`) from [GitHub Packages](https://github.com/orgs/scalar-labs/packages).
```
Comment (feeblefakie) [inline docs/getting-started-scalar-manager.md:16]:
```suggestion
* You need privileges to pull the Scalar Manager container (`scalar-manager`) from [GitHub Packages](https://github.com/orgs/scalar-labs/packages).
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left one minor suggestion.

File: docs/getting-started-scalar-manager.md (hunk 1 above)
Comment (kota2and3kan) [reply docs/getting-started-scalar-manager.md]: @feeblefakie  
Thank you for the suggestion!
I applied the suggestion.
